### PR TITLE
Mark `cuquantum` CUDA 11.8 migration done

### PR DESCRIPTION
**Pull request by jakirkham**
The CUDA 11.8 migrator for `cuquantum` was generated by the bot and merged late last week. However it appears the bot now notes `cuquantum` under error instead of complete. Adjust the metadata here to hopefully update the bot's status of `cuquantum`.

xref: https://github.com/conda-forge/cuquantum-feedstock/pull/49

### CODE REVIEW

**Comment by jakirkham** [comment]
@beckermr could you please take a look?

File: pr_info/e/b/b/d/9/cuquantum.json [hunk 1]
```diff
@@ -138,7 +138,8 @@
   },
   {
    "PR": {
-    "__lazy_json__": "pr_json/1540356936.json"
+    "number": null,
```
Comment (beckermr) [inline pr_info/e/b/b/d/9/cuquantum.json]: Can you add the number here please?

Comment (jakirkham) [reply pr_info/e/b/b/d/9/cuquantum.json]: Like so?

```suggestion
    "number": 49,
```

**Comment by jakirkham** [comment]
Thanks Matt! 🙏

**Comment by jakirkham** [comment]
Looks like that fixed it! 😄

**Pull request merged**